caffe2/core/blob.h

```diff
@@ -52,7 +50,7 @@ inline Tensor* BlobGetMutableTensor(
   VLOG(1) << "Create new mutable object " << TypeMeta::TypeName<Tensor>()
           << " dims: " << dims;
   // << " options: " << options; (operator<< for Options is in at:: now)
-  return blob->Reset<Tensor>(new Tensor(empty(dims, options)));
+  return blob->Reset<Tensor>(new Tensor(caffe2::empty(dims, options)));
```

Comment:
Not for this PR, but this code really wants to make me gouge my eyes out.

Reply:
need to change Tensor* to Tensor?